src/Prefixed/README.md

```diff
@@ -6,7 +6,7 @@
 [![Monthly Downloads](https://poser.pugx.org/cache/prefixed-cache/d/monthly.png)](https://packagist.org/packages/cache/prefixed-cache)
 [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
 
-This is a decorator for a PSR-6 hierarchical cache. It will allow you do use namespaces.
+This is a decorator for a PSR-6 hierarchical cache. It will allow you to use namespaces.
```

Comment:
Thank you. 

But reading this line I see that it is a copy-paste issue. It should be:

`... It will allow you to prefix all cache items with a predefined key.`